karma/module.py

```diff
@@ -407,15 +407,31 @@ async def karma_vote(
 
         # Fetch updated message with the votes
         vote_message = await vote_message.channel.fetch_message(vote_message.id)
+        total_reactions = 0
+
         for reaction in vote_message.reactions:
             votes[reaction.emoji] = reaction.count - 1
+            total_reactions += reaction.count - 1
 
         log_message: str = (
             f"Karma vote over emoji '{emoji_name}' ended: "
             + ", ".join(f"{v}x {emoji_labels[k]}" for k, v in votes.items())
             + "."
         )
 
+        if total_reactions < voter_limit:
```

Comment:
Uhm, why don't you just add up the individual emoji counts, why are are you creating another variable?